pdns/lua-recursor4.cc

```diff
@@ -365,6 +365,12 @@ RecursorLua4::RecursorLua4(const std::string& fname)
   d_lw->registerMember<bool (DNSQuestion::*)>("variable", [](const DNSQuestion& dq) -> bool { return dq.variable; }, [](DNSQuestion& dq, bool newVariable) { dq.variable = newVariable; });
   d_lw->registerMember<bool (DNSQuestion::*)>("wantsRPZ", [](const DNSQuestion& dq) -> bool { return dq.wantsRPZ; }, [](DNSQuestion& dq, bool newWantsRPZ) { dq.wantsRPZ = newWantsRPZ; });
 
+  d_lw->registerMember<const dnsheader (DNSQuestion::*)>("dh",
+    [](const DNSQuestion& dq) -> const dnsheader {
+      return *dq.dh;
```

Comment:
I know I wrote that code, but we should probably check that `dq.dh` is not `NULL` before de-referencing it. Sorry!